ODIN-EE

liebestraumm opened the pull request:
Event extraction on EE. Only works with basic sentences

Commit:
Merge pull request #35 from infosci-qut/develop

Develop

Commit:
Merge pull request #36 from infosci-qut/develop

Develop

Commit:
Merge pull request #37 from infosci-qut/develop

dependency updates

vincent659 committed:
graph controller comment

vincent659 committed:
graph controller comment

vincent659 committed:
graph controller comment

vincent659 committed:
graph controller comment

vincent659 committed:
Tap_server1

vincent659 committed:
Second update of the app folder

vincent659 committed:
Finalising app class documentation

vincent659 committed:
Tap_Server_2

Commit:
Merge pull request #38 from vincent659/app.classDoc

App.class doc

Commit:
Merge pull request #39 from vincent659/tap_server_1

Tap server 1

Commit:
Merge pull request #40 from vincent659/app.classDoc_3

App.class_1

Commit:
Merge pull request #41 from vincent659/Tap_Server_2

Tap server 2

vincent659 committed:
Merge pull request #1 from infosci-qut/develop

Develop

vincent659 committed:
Tap

vincent659 committed:
Merge remote-tracking branch 'origin/develop' into develop

andreyinkin committed:
Merge pull request #1 from heta-io/master

Merge latest TAP release into the develop branch

andreyinkin committed:
Fixed athanor URL error: convert option to string

andreyinkin committed:
Merge pull request #2 from infosci-qut/develop

Develop

Commit:
Merge pull request #43 from uts-cic/develop

Develop

Commit:
Merge pull request #42 from vincent659/develop

Develop

Commit:
Merge pull request #44 from infosci-qut/develop

Develop

andrewresearch reviewed (changes requested):
Carlos, this needs to be a PR against the QUT version of TAP: https://github.com/infosci-qut/tap

liebestraumm committed:
Event Extractor. Shows results on console.
- 3 yml rules (master, events and entities) in rules folder
- New controller (OdinController)
- New view (Test). This view prints results on console, for now
- New route (localhost/test) which is used temporarily for tests. This will be changed soon.
To see the results, use this url: localhost:9000/test

Signed-off-by: Carlos <carlosarturo.agelvisvergara@connect.qut.edu.au>

Commit:
Merge pull request #54 from liebestraumm/develop

Event Extractor. Shows results on console.

andrewresearch commented:
Closing old PR
